### DISCUSSION TITLE
Outline like RStudio

### DISCUSSION THREAD

**Discussion by lixiang117423** [posit-dev/positron#4080] [Feedback]
I have completely switched from RStudio to Positron and I am very satisfied! One last small request (or question): Can I display the Outline on the right side like RStudio? The current floating Outline is not very convenient for me.

Thank you!

Xiang

**Comment by juliasilge** [comment]
Do you mean this outline?

![test_qmd_—_scratch](https://github.com/user-attachments/assets/7a7aa5a9-5865-4e35-813f-c41c343e9ab2)

You can switch the positions of the primary and secondary side bars so this is on the right by choosing "Move Primary Side Bar Right", or you can grab it and move it where you prefer, like here:

![moved](https://github.com/user-attachments/assets/cacf1392-a027-4f9a-a200-88c7c6d5cb08)

If you ever get into a bad situation with your layout, go back to one of the defaults like "Stacked Layout".

**Reply by juliasilge** [reply]
Also related to outlines, be aware of #3822.

**Reply by lixiang117423** [reply]
YeeeeeS :)

This is what I wanted, thank you very much!

**Comment by wang-jie-98** [comment]
How can we achieve the same functionality in a Python script?

**Reply by juliasilge** [reply]
In a Python script, the outline will get generated something like this:

<img width="1580" height="1060" alt="Screenshot_2025-10-27_at_8_22_58 AM" src="https://github.com/user-attachments/assets/7f220214-8fb9-45a7-93be-60f8be03d7dc" />
